GoogleForCreators/web-stories-wp
# Redesign: switch the color scheme

## Context
Does the following:
- changes the old theme to `theme.DEPRECATED_THEME` and starts using the new design system under `theme`.
- Switches some main colors to the new color scheme already, such as background colors and some font colors.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Open the editor
2. See that things look "normal"


### QA
Even though this PR has some visual changes, there is no point doing QA for it since it mainly changes to the design-system and the specific sections of the workspace will be all addressed properly in separate issues.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6209
